auth/proxy/app.ts

```diff
@@ -1,126 +1,26 @@
-import { APIGatewayProxyResultV2, APIGatewayProxyEventV2, APIGatewayProxyEventHeaders } from 'aws-lambda';
+import { APIGatewayProxyResultV2 } from 'aws-lambda';
```

Comment:
Feedback was that this shouldn't have the createLambda function as well as injecting dependencies. Inspired by [DI](https://github.com/govuk-one-login/mobile-platform-back/blob/main/backend-api/src/authorizers/appCheck/appCheckHandler.ts#L30)